compiler/src/org.graalvm.compiler.asm.amd64/src/org/graalvm/compiler/asm/amd64/AMD64Assembler.java

```diff
@@ -72,6 +72,10 @@
 import jdk.vm.ci.code.TargetDescription;
 import jdk.vm.ci.meta.PlatformKind;
 
+// @formatter:off
```

Comment:
This effectively turns off formatting for the whole file. Please remove it. The pattern for disabling/re-enabling formatting is:
```
// @formatter:off
<code that you don't want reformatted>
// @formatter:on
```

As I mentioned offline, the Eclipse re-formatting of commented boolean constant arguments in calls to the `AMD64InstructionAttr` is unfortunate. @mur47x111 is converting these booleans to a set of enum values.